utils/validator.js

```diff
@@ -48,3 +49,26 @@ export const isAdmin = () => {
         next();
     };
 };
+
+export const isNotParents = () => {
+    return async (req, res, next) => {
+        const user = await User.findById(req.user._id);
+        if (user.roles.includes("guardian")) {
+            return fMsg(res, "Unauthorized", "You are a guardian");
+        }
+        next();
+    }
+}
+
+export const isEditorStranger = () => {
```

Comment:
Like above i am not sure too but we will have to clarify with the sayr for permission level authorization likee.. but code is great nice idea